### PR TITLE
Integrate antsibull-docutils into CI and build scripts and instructions

### PR DESCRIPTION
**Pull request by felixfontein**
Prepares for https://github.com/ansible-community/antsibull-changelog/pull/174.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,23 +35,25 @@ Install and run `nox` to run all tests. That's it for simple contributions!
...
 
-antsibull-docs depends on the sister antsibull-core and antsibull-docs-parser projects.
-By default, `nox` will install a development version of these projects from
-Github.
-If you're hacking on antsibull-core and/or antsibull-docs-parser alongside antsibull-docs,
-nox will automatically install the projects from `../antsibull-core` and
-`../antsibull-docs-parser` when running tests if those paths exist.
+antsibull-docs depends on the sister antsibull-changelog, antsibull-core, and
+antsibull-docs-parser projects. By default, `nox` will install a development version of
+these projects from Github.
```
Comment (gotmax23) [inline README.md:42]: I wrote this section originally to adhere to sembr.org.

```suggestion
antsibull-docs depends on the sister antsibull-changelog, antsibull-core, and
antsibull-docs-parser projects.
By default, `nox` will install a development version of these projects from Github.
```

File: noxfile.py [hunk 2]
```diff
@@ -96,7 +101,7 @@ def test(session: nox.Session):
 
 @nox.session
 def coverage(session: nox.Session):
-    install(session, "coverage[toml]")
+    install(session, "coverage[toml]", *other_antsibull())
```
Comment (gotmax23) [inline noxfile.py:104]: `*other_antsibull()` shouldn't be needed just to run coverage.

File: noxfile.py [hunk 3]
```diff
@@ -115,7 +120,7 @@ def lint(session: nox.Session):
 
 @nox.session
 def formatters(session: nox.Session):
-    install(session, ".[formatters]")
+    install(session, ".[formatters]", *other_antsibull())
```
Comment (gotmax23) [inline noxfile.py:123]: Same here.

File: noxfile.py (hunk 2 above)
Comment (felixfontein) [reply noxfile.py:104]: Ah sorry, I thought it also says `.[coverage]` here as in antsibull-changelog.

File: README.md (hunk 1 above)
Comment (felixfontein) [reply README.md]: ecc0c55f74c0a7cff7d961b53ca1e11b2e51b32f.

File: noxfile.py (hunk 2 above)
Comment (felixfontein) [reply noxfile.py]: 52a8e59af928a77f1e12555f963b5029b2167215.

**Comment by felixfontein** [comment]
I've marked this back to ready for review since it's needed for https://github.com/ansible-community/antsibull-changelog/pull/174.

**Comment by gotmax23** [comment]
I'll try to get to this later today.

**Comment by gotmax23** [comment]
I'll try to get to this later today.

**Comment by felixfontein** [comment]
I agree. Right now it only uses it indirectly, but if we want to add MarkDown output to antsibull-docs at some point, it will be useful (and then it definitely should become a direct dependency). Or if antsibull-docutils has a RST writer, then we could allow collections to have extra docs in MarkDown.

**Comment by felixfontein** [comment]
@gotmax23 thanks for this one as well!

**Pull request merged**